src/hyperscan.mk

```diff
@@ -9,6 +9,7 @@ $(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $($(PKG)_SUBDIR).tar.gz
 $(PKG)_URL      := https://github.com/01org/$(PKG)/archive/v$($(PKG)_VERSION).tar.gz
 $(PKG)_DEPS     := gcc boost
+# $(PKG)_NATIVE_DEPS := ragel
```

Comment:
I'll submit a PR for #516 soon, native deps will look something like `$(BUILD)~ragel` (see [gcc](https://github.com/tonytheodore/mxe/blob/2f1d18539ddb333f9595c730e75fba0669d76b77/src/gcc.mk#L12) or [qbittorrent](https://github.com/tonytheodore/mxe/blob/2f1d18539ddb333f9595c730e75fba0669d76b77/plugins/apps/qbittorrent.mk#L11)).